### ISSUE TITLE
Traduction mod better sorting to Spanish of Spain

### ISSUE THREAD

**Issue by Qkiiee** [ChobitsCrazy/BetterSorting#72]
Hello! I am contacting you to tell you that I have translated your mod into Spanish of Spain. If you want the translation, I'll send it to you without any problem. I was going to use it for my own use, but I thought it was better to send it to you if you want it. 
Greetings!^^

**Comment by ChthonicKnight** [comment]
Yes you can send it here and I'll take a look to add it for the next update.
